scripts/sanitycheck

```diff
@@ -2338,11 +2342,19 @@ def parse_arguments():
         "that have that value defined. For the <config> case, match "
         "defconfig that have that value assigned to any value. "
         "Prepend a '!' to invert the match.")
+
     parser.add_argument(
         "-s", "--test", action="append",
         help="Run only the specified test cases. These are named by "
         "<path to test project relative to "
         "--testcase-root>/<testcase.yaml section name>")
+
+    parser.add_argument(
+        "--sub-test", action="append",
+        help="Run only the specified sub-test cases and its parent. These are named by "
+        "test case name appended by test function, i.e. kernel.mutex.semaphore_init."
+        )
```

Comment:
I think `kernel.mutex.mutex_lock_unlock` would be better (or any other test funtion) instead of `kernel.mutex.semaphore_init` as we don't have any such test function, `test_semaphore_init` with test name `kernel.mutex`.